pages/download.vue

```diff
@@ -29,10 +29,20 @@
                     </b-button>
 
                 </b-alert>
-                <b-alert v-model="outputIsValid" variant="success">
+                <b-alert v-model="outputIsValid" variant="success" :show="dictionaryCollapsed">
                     <h1>🎉 Congratulations!</h1>
                     You have successfully created an <a href="https://neurobagel.org/dictionaries/" target="_blank">neurobagel annotated .json data dictionary</a>.
                 </b-alert>
+                <h3>Data Dictionary</h3>
```

Comment:
This section still shows up every time. I think we would want it to only show up if the dictionary is valid. A test could be useful to check this.